templates/winogrande/winogrande_debiased/templates.yaml

```diff
@@ -0,0 +1,47 @@
+dataset: winogrande
+subset: winogrande_debiased
+templates:
+  0cc669d0-da87-411d-9e1c-8e102da63c2c: !Template
+    id: 0cc669d0-da87-411d-9e1c-8e102da63c2c
+    jinja: "{{sentence}}\nReplace the _ in the above sentence with the correct option:\
+      \ \nOption1: {{option1}}\nOption2: {{option2}}\n|||\n{% if answer== 1 %}{{option1}}{%\
+      \ else%}{{option2}}{% endif %}"
+    name: Replace
+    reference: Replace with correct option
+  13b2dbe4-abf3-4b09-b7cb-459224881844: !Template
+    id: 13b2dbe4-abf3-4b09-b7cb-459224881844
+    jinja: 'The _ in the sentence below refers to {{option1}}. Answer True or False!
+
+      {{sentence}}|||
+
+      {% if answer== 1 %}True{% else%}False{% endif %}'
+    name: True or False
+    reference: True or False Type
```

Comment:
Can you point me to where these came from? The reference field is for citing where the prompt originally was proposed, not for a description, so we should remove these.

Reply:
Remove the descriptions from the reference fields.